send-reminder.py

```diff
@@ -32,7 +32,9 @@
 
 DESCRIPTION = '''{} as Triage Sheriff #1 this week, can you please take the lead to coordinate a date/time this week?
 
-For the latest bug nominations, please use https://mzl.la/perf-triage-query.'''
+For the latest bug nominations, please use https://mzl.la/perf-triage-query.
+
+Triage rotation: https://mozilla.github.io/perf-triage/'''
```

Comment:
Perhaps we could just link to the (new) wiki page, so it's easier to keep these links up to date? https://wiki.mozilla.org/Performance/Triage